javascript/javascript1/week1/housey-pricey.js

```diff
@@ -0,0 +1,39 @@
+// Housey pricey (A house price estimator)
+
+/* Two of your friends are considering buying a house, but cannot figure out what the right price should be. 
+Your friends know the width, the height and the depth of the house and the garden size. 
+Lets help them figure out if they paid too much */
+
+class House {
+    constructor(width, height, depth, gardenSizeInM2, houseCost) {
+        this.width = width;
+        this.height = height;
+        this.depth = depth;
+        this.gardenSizeInM2 = gardenSizeInM2;
+        this.houseCost = houseCost;
+    }
```

Comment:
You have good hand in js
Just one thing. Change hard coded values with variables inside of class.

Reply:
Thanks for your comment. Could you please give me an example of how I should do it because I couldn't figure it out accurately.